app/templates/labs.html

```diff
@@ -5,18 +5,72 @@
 
 {% block main %}
 <section class="card top-card">
-   <div class="content">
-       <h1 class="page-title">ADI Labs</h1>
-       <div class="section-header">
-           <h4>About</h4>
-           <span class="bar green"></span>
-       </div>
-   </div>
-   <div class="content">
-        <h3>Learn software engineering by working on apps used by thousands of Columbia students! ADI Labs meets once a week for four hours to develop software intended to make life easier at Columbia.
+     <div class="content">
+         <h1 class="page-title">ADI Labs</h1>
+         <div class="section-header">
+             <h4>About</h4>
+             <span class="bar green"></span>
+         </div>
+     </div>
+     <div class="content">
+        <h3>ADI Labs is your opportunity to learn everything you can’t learn in class, from software engineering to UI/UX design to product management. At ADI Labs, you will build real projects like Density by ADI that you will ship. Along the way you’ll receive personalized mentorship from peers that have worked at Google, Facebook, Microsoft, LinkedIn, Coursera, and others. You’ll also join an awesome community of builders that love to bring ideas to life as well as have a good time doing so.
         </h3>
         <h3>As a member of Labs, you’ll learn to contribute to open source projects and to take your programming beyond problem sets. Paired with one of our student mentors, you’ll learn software engineering techniques used in industry.</h3>
-   </div>
+    </div>
+</section>
+<section>
```

Comment:
yeah we're on time crunch though. i'll make an issue

On Mon, Sep 14, 2015 at 2:34 PM Nate <notifications@github.com> wrote:

> In app/templates/labs.html
> <https://github.com/adicu/adi-website/pull/240#discussion_r39429581>:
>
> >          </h3>
> >          <h3>As a member of Labs, you’ll learn to contribute to open source projects and to take your programming beyond problem sets. Paired with one of our student mentors, you’ll learn software engineering techniques used in industry.</h3>
> > -   </div>
> > +    </div>
> > +</section>
> > +<section>
>
> This all looks like it should be a template instead of raw HTML.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adicu/adi-website/pull/240/files#r39429581>.
>